### DISCUSSION TITLE
plan to use super-linter

### DISCUSSION THREAD

**Discussion by DanielRuf** [verdaccio/verdaccio#1851] [Ideas]
**Describe the solution you'd like**
I would like to try [super-linter](https://github.com/github/super-linter) to replace some linter setups to make it easier for us to lint different languages and file formats.

**Describe alternatives you've considered**
Using multiple linters with multiple configs.

**Additional context**

**Comment by juanpicado** [comment]
Go ahead for my side :) also looking forward ...  @sergiohgz thoughts? you are the linter master ... cc: @verdaccio/core-team @verdaccio/collaborators 

btw, do we have access?

any const to keep on mind?

**Comment by sergiohgz** [comment]
I don't see any benefits on our setup, but it could be great to have a PoC about 😉

**Comment by juanpicado** [comment]
I see the benefit in three areas which we are not covering much now:

* Yaml
* Dockerfile
* Markdown
